src/wpilib-ws-client.ts

```diff
@@ -44,28 +44,28 @@ export default class WPILibWebSocketClient extends WPILibWSInterface {
         }
 
         const url = `ws://${this._hostname}:${this._port}${this._uri}`;
-        this._ws = new WebSocket(url);
+        this._ws = new ReconnectingWebSocket(url);
 
-        this._ws.on("open", () => {
+        this._ws.addEventListener("open", () => {
             this._ready = true;
             console.log("WS Ready");
             this.emit("ready");
             this.emit("openConnection");
         });
 
-        this._ws.on("close", () => {
+        this._ws.addEventListener("close", () => {
             this._ready = false;
             console.log("WS Closed");
             this.emit("closeConnection");
         });
 
-        this._ws.on("error", (code: number, reason: string) => {
```

Comment:
Wasn't sure what was the best way to handle the difference here since ReconnectingWebSocket/browser WebSocket interface does not provide a code. If providing a code is still desired then maybe the "ws" package should be used for node environments and ReconnectingWebSocket for browser environments?